doc/introduction/operations.rst

```diff
@@ -142,6 +142,20 @@ Qubit observables
 
 :html:`</div>`
 
+.. _intro_ref_ops_measurements:
+
+Qubit measurements
+^^^^^^^^^^^^^^^^^^
```

Comment:
I feel like this will be a point of confusion to end-users - how will they distinguish between these mid-circuit projective measurements, and terminal measurements we already have?

Reply:
![image](https://user-images.githubusercontent.com/2959003/108061817-d15d7b00-7093-11eb-9eac-c4ba900b38d0.png)

Reply:
@josh146: This is a very good point. My personal opinion is that "Measurements" doesn't describe well the content inside this section of the documentation. Really it's more about circuit outputs, and how pennylane supports things like variances and probabilities. Additionally, I think that the name `Measure()` is perfect for the operation of measuring a wire mid circuit.

So my suggestion is that we consider changing the name "Measurements" in the documentation

Reply:
> So my suggestion is that we consider changing the name "Measurements" in the documentation

A good suggestion, my only hesitation is that it will be more invasive/work than just the title of the Measurements page unfortunately - the term 'measurements' (referring to post-processing terminal measurements) is used across the demos and API documentation as well.

Reply:
As a compromise, could we call it "Qubit projections"?